test-app/src/index.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+<head>
+  <title>createjs Accessibility Test Page</title>
+
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <style type="text/css">
+  canvas {
+    background-color: white;
+    outline: none;
+  }
+  </style>
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.20.2/TweenLite.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.20.2/TimelineMax.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.20.2/plugins/CSSPlugin.min.js"></script>
+  <script src="https://code.createjs.com/easeljs-0.8.2.min.js"></script>
+</head>
+<body>
+  <div id="cam-test"></div>
+  <canvas id="stage" width="800px" height="600px" tabindex="-1"></canvas>
+</body>
+</html>
```

Comment:
diff'ed `/test-app/src` folder against `/src` folder from `CurriculumAssociates/createjs-accessibility-tester` and verified contents are exactly the same